myapp/spec/system/tasks_spec.rb

```diff
@@ -34,6 +54,68 @@
 
   describe 'task creation' do
     before do
+      create_user
+      login
+    end  
+    it 'task created successfully & show flash message when task created', :aggregate_failure do
+        visit '/'
+        click_link('New task')
+        # expect(Task.all.length).to eq 0
```

Comment:
if no reason to put this comment for another developer, it's better to remove it 👍